### PR TITLE
Dev 2745 user last login admin page

### PR DESCRIPTION
**Pull request by harsha09**
#### What's this PR do?
Adds `last_login` field to the users table in Django Admin.

#### Why are we doing this? How does it help us?
Admin to know the context of user in on-boarding process.

#### How should this be manually tested? Please include detailed step-by-step instructions.
Login to Django Admin -> Go to `Users` section -> `Last Login` column should be displayed.
Click on a user in user list -> `Last Login` field displayed which is read-only.

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
Changes only in Django Admin.

#### Have automated unit tests been added? If not, why?
Added.

#### How should this change be communicated to end users?
NA

#### Are there any smells or added technical debt to note?
NA

#### Has this been documented? If so, where?
Not yet

#### What are the relevant tickets? Add a link to any relevant ones.
[DEV-2745]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
Adding one environment variable `UPDATE_LAST_LOGIN` to update the user last_login in `SIMPLE_JWT` auth. By default it is set to `True`.

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
NA

[DEV-2745]: https://news-revenue-hub.atlassian.net/browse/DEV-2745?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/users/admin.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def role_assignment(self, obj):
             None,
             {
                 "fields": (
+                    "last_login",
                     "email",
```
Comment (leo-dcfa) [inline apps/users/admin.py:30]: I know this is a small change but a unit test wouldn't hurt 
See example here:

https://github.com/newsrevenuehub/rev-engine/blob/main/apps/users/tests/test_admin.py#L33

I would add a unit test for both detail as well as list display

**Review by leo-dcfa — CHANGES_REQUESTED** [review]
Small change requested

File: apps/users/admin.py (hunk 1 above)
Comment (harsha09) [reply apps/users/admin.py:30]: addressed using 472dec2941a358700f53016787113e81ac0c5de5

**Review by leo-dcfa — APPROVED** [review]

**Review by leo-dcfa — APPROVED** [review]
Test performed:

Created new user leo.alves+dev-2745@fundjournalism.org
Logged in through sign-in page (not Django Admin)
Last login was updated

**Review by x110dc — APPROVED** [review]

**Pull request merged**